app/controllers/private/projects_controller.rb

```diff
@@ -41,6 +41,15 @@ def destroy
     redirect_to private_projects_url, notice: t('.notice')
   end
 
+  def slug
+    response = SlugChecker.call(
```

Comment:
Аналогично вот этому пункту стоит сделать: https://github.com/rubocop-hq/ruby-style-guide#indent-conditional-assignment

```suggestion
    response =
      SlugChecker.call(
```